js/planning.js

```diff
@@ -650,6 +622,30 @@ var GLPIPlanning  = {
         //window.calendar = calendar; // Required as object is not accessible by forms callback
         GLPIPlanning.calendar.render();
 
+        // IMPORTANT: This event source was moved here, after the calendar's render() call,
+        // to prevent an automatic AJAX request to the events URL during FullCalendar's initialization.
+        // And another one during the calendar's changeView() call.
+        // By adding it manually after rendering, only one call is done.
+        this.calendar.addEventSource({
```

Comment:
Won't this mean that event wont be loaded if the view change (invalid view) before this handler is added ?

Reply:
The handler is added to the `calendar` instance (and not the view), so this has no impact on the loading of events when the view changes.